README.md

```diff
@@ -1,20 +1,20 @@
-### OpenCV: Open Source Computer Vision Library
+## OpenCV: Open Source Computer Vision Library
 
-#### Resources
+### Resources
 
 * Homepage: <http://opencv.org>
 * Docs: <http://docs.opencv.org/master/>
 * Q&A forum: <http://answers.opencv.org>
 * Issue tracking: <https://github.com/opencv/opencv/issues>
```

Comment:
I can use the markdown syntax for these links too, but they look fine the way they are. Your suggestion?